modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
+
+Use {pipelines-shortname} to run pods in the user namespace.
+
+.Prerequisites
+
+* Pass the annotations that CRIO expects.
+
+.Procedure
+
+. Use the `oc edit clustertask buildah` command to edit the buildah cluster task.
+
+. Add the annotations for all users.
```

Comment:
Do you think users might need an example of what adding the annotations looks like?

Reply:
The main concern I have is whether lines 18 (Add the annotations for all users)  and 20 (Export the cluster task as a task to the namepsace) should be separate procedure modules. I'm checking with @abrennan89 to see what she thinks.

Reply:
> Something I noticed trying to check this, it doesn't look like the module has a list of assemblies it's included in, FYI

I suggested [the same in this comment](https://github.com/openshift/openshift-docs/pull/44733/files#r897035726). I think @HarshCasper is fixing items and marking them as resolved, but hasn't pushed his latest updates to this PR yet.

Reply:
Need SME input here.

Reply:
@HarshCasper is the need for input about which command to use to edit the global task? as @abrennan89 suggested if the user want to modify the task for every users on clusters (and have the rights for it) she/he can edit the clustertasks directly i.e: `oc edit clustertasks buildah`